assets/js/components/DeregistrationModal/DeregistrationModal.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { EOS_CLEANING_SERVICES } from 'eos-icons-react';
+
+import Modal from '@components/Modal';
+import Button from '@components/Button';
+
+function DeregistrationModal({
+  hostName,
+  isOpen = false,
+  onCleanUp,
+  onClose = () => {},
```

Comment:
I will suggest to not use any default for the onClose, we don't have any default for onCleanUp, I don't see the reason for having the onclose defined with default, this could led to an unexpected flow

Reply:
I will also suggest to use onCancel intestead of onClose, we have onCleanUp because the cta is cleanup, we have cancel as cta button, and onClose, move to onCancel for consistency